Project/src/MakeCall/CallCard.js

```diff
@@ -669,7 +682,10 @@ export default class CallCard extends React.Component {
         try {
             this.meetingReaction?.sendReaction(reactionMessage);
         } catch (error) {
+            // Surface the error 
             console.error(error);
+            messageBarText = JSON.stringify(error);
```

Comment:
Can you add a screen shot to this PR on how this messageBarText shows up on the UI.

Reply:
screenshot added in the PR. Thanks :)